### DISCUSSION TITLE
🏄‍♀️ Surf The World 🤙

### DISCUSSION THREAD

**Discussion by helenamerk** [supabase/supabase#216] [Show and tell]
tldr we built **[a chrome extension](bit.ly/stw-chrome)** that lets you bump into friends online 👯‍♀️

## **the motivation?** 🤔

Serendipity is an amazing feeling. When you run into a friend at Trader Joes, or your classmate in a cafe, it sparks conversation and creativity. With the pandemic this happens less!! Let's bring serendipity to the internet 💪

## 🙋🏼‍♀️ **the ask?** 

looking for feedback! add a friend by username and see if you run into eachother. lmk about any feature requests :)

## 🙅🏼‍♀️ **my privacy???** 

we do not store or sell any of your data. no, not even your browsing history.

your domain will be shared only with people you add as friends AND happen to be on the same domain. (ie not which github repo you are on or twitter you are stalking. only that you are on github or twitter.) 

you can pause the extension at any point for any number of hours. 

##  😍 **how do I start?**

[Check out our demo on youtube](https://www.youtube.com/watch?v=gcyzE6KXkR0)

Or install now: **[Surf The World Extension](bit.ly/stw-chrome)**

## 🥰 thank you to supabase!! 

Supabase makes this all possible! We use supabase realtime to send updates across browsers. The backend and supabase integration took almost no time to set up... Meanwhile the chrome extension portion was a pain (@brianli101 can attest). 

Shoutout to @awalias and @kiwicopple for making supabase a reality and being our first fans 🙌

![119892999_2775696776039094_645635906453438395_o](https://user-images.githubusercontent.com/7145275/94216287-3c67a480-fe9c-11ea-810d-e52616e199aa.jpg ) 
![120014631_2775696789372426_2890184444502378860_o](https://user-images.githubusercontent.com/7145275/94216362-6c16ac80-fe9c-11ea-8d4a-9fcd5ba3c8eb.jpg)
![119883072_2775696792705759_8996692967259767431_o](https://user-images.githubusercontent.com/7145275/94216413-83559a00-fe9c-11ea-97c3-f1c3710723bf.jpg)

**Comment by katzchen0713** [comment]
Awesome and so creative!! Have been using it for a while and the serendipity brings a little spice to my pretty mundane Zoom life nowadays. Love it!

**Comment by BrianLi101** [comment]
We love building with Supabase! As a visual person, it makes it so much easier for me to manage data when developing so that I'm not slowed down by complicated technical details :)

**Comment by ali-aka-ahmed** [comment]
Very cool! Love the design and can appreciate how hard chrome extensions are <3
